### DISCUSSION TITLE
Document Variables Available For Config Template Editing

### DISCUSSION THREAD

**Discussion by Kurt-von-Laven** [renovatebot/renovate#18572] [Request Help] [answered]
### What would you like Renovate to be able to do?

The [Config Template Editing docs](https://github.com/renovatebot/renovate/blob/a35b4ae7094f06983729a42a12052d72d1d168d7/docs/usage/configuration-templates.md) help me understand which setting I should be configuring, but they don't help me understand what value I should set it to.

### If you have any ideas on how this should be implemented, please tell us here.

The default value of [`commitMessageExtra`](https://docs.renovatebot.com/configuration-options/#commitmessageextra) (`with {{newName}} {{#if isMajor}}{{{prettyNewMajor}}}{{else}}{{#if isSingleVersion}}{{{prettyNewVersion}}}{{else}}{{{newValue}}}{{/if}}{{/if}}`) reveals some of the variables that can be used, but presumably there are additional variables available. Consequently, I made a [completely failed attempt at adding the current version to the commit message](https://github.com/ScribeMD/.github/blob/4e213374619b2f010912edc89bfcab1f23464f94/default.json#L14). Maybe I should be using some of the variables defined in [`lib/modules/manager/types.ts`](https://github.com/renovatebot/renovate/blob/a35b4ae7094f06983729a42a12052d72d1d168d7/lib/modules/manager/types.ts), but I haven't found a comprehensive list of what is available even in the source code. See ScribeMD/.github#97 for my stab in the dark attempt at correcting `commitMessageTopic`.

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment, marked as answer]
https://docs.renovatebot.com/templates/

**Reply by Kurt-von-Laven** [reply]
Ah hah; thank you! This would be worth linking to from the fields that support templates.
